chpl/chpl-api/src/main/java/gov/healthit/chpl/questionableActivity/CertificationResultQuestionableActivityProvider.java

```diff
@@ -78,31 +99,39 @@ public QuestionableActivityCertificationResultDTO checkGapUpdated(
+     * @param origCertResult original certification criteria result
+     * @param newCertResult new certification criteria result
+     * @return list of added measures
+     */
     public List<QuestionableActivityCertificationResultDTO> checkG1MacraMeasuresAdded(
-            CertificationResult origCertResult, CertificationResult newCertResult) {
-        
-        List<QuestionableActivityCertificationResultDTO> addedMacras = new ArrayList<QuestionableActivityCertificationResultDTO>();
-        if ((origCertResult.getG1MacraMeasures() == null || origCertResult.getG1MacraMeasures().size() == 0) && 
-                newCertResult.getG1MacraMeasures() != null && newCertResult.getG1MacraMeasures().size() > 0) {
+            final CertificationResult origCertResult, final CertificationResult newCertResult) {
+
+        List<QuestionableActivityCertificationResultDTO> addedMacras
+        = new ArrayList<QuestionableActivityCertificationResultDTO>();
+        if ((origCertResult.getG1MacraMeasures() == null || origCertResult.getG1MacraMeasures().size() == 0)
+                && newCertResult.getG1MacraMeasures() != null && newCertResult.getG1MacraMeasures().size() > 0) {
             //all the newcert g1 macras are "added"
-            for(MacraMeasure newMacra : newCertResult.getG1MacraMeasures()) {
+            for (MacraMeasure newMacra : newCertResult.getG1MacraMeasures()) {
                 QuestionableActivityCertificationResultDTO activity = new QuestionableActivityCertificationResultDTO();
                 activity.setBefore(null);
                 activity.setAfter(newMacra.getAbbreviation());
                 addedMacras.add(activity);
             }
-        } else if (origCertResult.getG1MacraMeasures() != null && origCertResult.getG1MacraMeasures().size() > 0 && 
-                newCertResult.getG1MacraMeasures() != null && newCertResult.getG1MacraMeasures().size() > 0) {
+        } else if (origCertResult.getG1MacraMeasures() != null && origCertResult.getG1MacraMeasures().size() > 0
+                && newCertResult.getG1MacraMeasures() != null && newCertResult.getG1MacraMeasures().size() > 0) {
             for (MacraMeasure newMacra : newCertResult.getG1MacraMeasures()) {
                 boolean hasMatch = false;
                 for (MacraMeasure origMacra : origCertResult.getG1MacraMeasures()) {
-                    if(newMacra.getId().longValue() == origMacra.getId().longValue()) {
+                    if (newMacra.getId().longValue() == origMacra.getId().longValue()) {
                         hasMatch = true;
                     }
                 }
-                if(!hasMatch) {
-                    QuestionableActivityCertificationResultDTO activity = new QuestionableActivityCertificationResultDTO();
+                if (!hasMatch) {
+                    QuestionableActivityCertificationResultDTO activity
+                    = new QuestionableActivityCertificationResultDTO();
```

Comment:
Should this be indented?  https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent